src/platformio.ini

```diff
@@ -79,12 +79,13 @@ extra_scripts = ${common_env_data.extra_scripts}
 lib_deps = NeoPixelBus
 
 [env:TX_TTGO_LORA_V1]
-platform = espressif32@1.12.0
```

Comment:
Removing version control is going to catch us out in the future.

Reply:
I'll add it back, was comparing code sizes